OSDOCS-5280: Add release note for multiple external gateway support (#61112)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

[id="ocp-4-14-networking"]
=== Networking

[id="ocp-4-14-multiple-external-gateway-support-ovn-kubernetes-network-plugin"]
==== Multiple external gateway support for the OVN-Kubernetes network plugin

The OVN-Kubernetes network plugin supports defining additional default gateways for specific workloads. Both IPv4 and IPv6 address families are supported. You define each default gateway with the `AdminPolicyBasedExternalRoute` object, where you can specify two types of next hops, static and dynamic:

- Static next hop: One or more IP addresses of external gateways
- Dynamic next hop: A combination of pod and namespace selectors for pod selection, and a network attachment definition name previously associated with the selected pods

> The next hops that you define are scoped by a namespace selector that you specify so that you can use the external gateway for specific workloads.

Author: @jordigilh, makes sense to emphasize, although we don't generally use bold or italics in the release notes. Including the text might be enough. Then we hopefully cover this well enough in the feature documentation as well.

> The next hops that you define are scoped by a namespace selector that you specify so that you can use the external gateway for specific workloads.

Reviewer: Makes sense and sorry about the bolt font. I did not mean to add it to the doc as such, just to visually identify the part of the sentence that I was referring to.

[id="ocp-4-14-networking"]
=== Networking

[id="ocp-4-14-multiple-external-gateway-support-ovn-kubernetes-network-plugin"]
==== Multiple external gateway support for the OVN-Kubernetes network plugin

> The OVN-Kubernetes network plugin supports defining additional default gateways for specific workloads. Both IPv4 and IPv6 address families are supported. You define each default gateway with the `AdminPolicyBasedExternalRoute` object, where you can specify two types of next hops, static and dynamic:

Reviewer 2: ```suggestion
The OVN-Kubernetes network plugin supports defining additional default gateways for specific workloads. Both IPv4 and IPv6 address families are supported. You define each default gateway using the `AdminPolicyBasedExternalRoute` object, in which you can specify two types of next hops, static and dynamic:
```

- Static next hop: One or more IP addresses of external gateways

> - Dynamic next hop: A combination of pod and namespace selectors for pod selection, and a network attachment definition name previously associated with the selected pods

Reviewer 2: ```suggestion
- Dynamic next hop: A combination of pod and namespace selectors for pod selection, and a network attachment definition name previously associated with the selected pods.
```

> The next hops that you define are scoped by a namespace selector that you specify so that you can use the external gateway for specific workloads that match the namespace selector.

Reviewer 2: ```suggestion
The next hops that you define are scoped by a namespace selector that you specify. This allows you to use the external gateway for specific workloads that match the namespace selector.
```
